spec.html

```diff
@@ -14752,7 +14752,7 @@ <h1>
 
     <emu-clause id="sec-typedarray-exotic-objects" oldids="sec-integer-indexed-exotic-objects">
       <h1>TypedArray Exotic Objects</h1>
-      <p>A TypedArray is an exotic object that performs special handling of integer index property keys.</p>
+      <p>A TypedArray is an exotic object that performs special handling of property keys that are canonical numeric strings, using the subset that are integer indices to index elements of uniform type and enforcing the invariant that the remainder are absent without incurring prototype chain traversal.</p>
```

Comment:
```suggestion
      <p>A TypedArray is an exotic object that performs special handling of property keys that are canonical numeric strings, using the subset that are in-bounds integer indices to index elements of uniform type and enforcing the invariant that the remainder are absent without incurring prototype chain traversal.</p>
```
maybe? There's not really a difference between out-of-bounds integer indices and non-integer numeric strings from the perspective of the spec machinery.

Reply:
Yeah, I like that. Done.